hubspot3/engagements.py

```diff
@@ -64,21 +64,39 @@ def update(self, key, data=None, **options):
             "engagements/{}".format(key), data=data, method="PUT", **options
         )
 
-    def get_all(self, **options):
+    def get_all(self, limit=-1, **options):
+        engagement_generator = self.get_all_as_generator(limit=limit,
```

Comment:
Do we have extra_properties for engagements? Even if we are not going to use them, the could should be able to handle them.

Reply:
There are no property choices in the engagements, the only options are items per page and offset

https://developers.hubspot.com/docs/methods/engagements/get-all-engagements

Reply:
Great!